src/main.cpp

```diff
@@ -3731,34 +3739,7 @@ bool CheckBlock(const CBlock& block, CValidationState& state, bool fCheckPOW, bo
             }
         }
     } else {
-        LogPrintf("CheckBlock(DASH) : skipping transaction locking checks\n");
-    }
-
-    // DASH : CHECK MASTERNODE PAYMENTS OR SUPERBLOCKS
-
-    CBlockIndex* pindexPrev = chainActive.Tip();
-    if(pindexPrev != NULL)
-    {
-        int nHeight = 0;
-        if(pindexPrev->GetBlockHash() == block.hashPrevBlock)
-        {
-            nHeight = pindexPrev->nHeight+1;
-        } else {
-            // IF WE CAN'T FIND A SPECIFIC MATCH
-            BlockMap::iterator mi = mapBlockIndex.find(block.hashPrevBlock);
-            if (mi != mapBlockIndex.end() && (*mi).second)
-                nHeight = (*mi).second->nHeight+1;
-        }
-
-        if(nHeight != 0){
-            if(!IsBlockPayeeValid(block.vtx[0], nHeight))
-            {
-                mapRejectedBlocks.insert(make_pair(block.GetHash(), GetTime()));
-                return state.DoS(100, error("CheckBlock() : Couldn't find masternode or superblock payments"));
-            }
-        } else {
-            LogPrintf("CheckBlock() : WARNING: Couldn't find previous block, skipping IsBlockPayeeValid(!)\n");
-        }
```

Comment:
it's over there https://github.com/dashpay/dash/pull/951/files#diff-7ec3c68a81efff79b6ca22ac1f1eabbaR2779